test/SmokeTests/OOProcSmokeTests/DotNetIsolated/host.json

```diff
@@ -7,5 +7,14 @@
         "excludedTypes": "Request"
       }
     }
-  }
+  },
+  "extensions": {
+    "durableTask": {
+      "storageProvider": {
+        "maxQueuePollingInterval": "00:00:01",
+        "controlQueueVisibilityTimeout": "00:01:00"
+      }
+    }
+  },
+  "functionTimeout": "00:00:30"
```

Comment:
We don't time out sooner because our OOM-based test needs time to run out of memory.